README.md

```diff
@@ -22,7 +22,4 @@ Tutorial will be helpful to you for customizing template-base!)
 
```

Comment:
@daisukenishino 

Please write the usage for users who use Visual Studio 2013 or earlier.